ui-tests/src/test/java/io/syndesis/qe/pages/SyndesisPageObject.java

```diff
@@ -107,6 +107,17 @@ public SelenideElement getElementByClassName(String elementClassName) {
 		return this.getRootElement().find(By.className(elementClassName));
 	}
 
+	public SelenideElement getElementByLocator(By elementLocator) {
+		log.info(String.format("searching for element %s", elementLocator.toString()));
```

Comment:
You don't need String.format in log message, just `log.info("searching for element {}", elementLocator.toString());`